zerver/lib/push_notifications.py

```diff
@@ -922,13 +922,7 @@ def process(elem: lxml.html.HtmlElement) -> str:
         return plain_text
 
     if settings.PUSH_NOTIFICATION_REDACT_CONTENT:
-        return (
-            "*"
-            + _(
-                "This organization has disabled including message content in mobile push notifications"
-            )
-            + "*"
-        )
+        return "*" + _("New message") + "*"
```

Comment:
Have you checked whether this actually appears in italics?

Reply:
It does not appear in italics. I'd left the asterisks in to follow previous convention but I do think having them here is redundant if just plain text. I'll update this PR based on the [CZO solution discussion](https://chat.zulip.org/#narrow/stream/49-development-help/topic/Potential.20solutions.20to.20issue.20.2329152/near/1762860) and your feedback.